integration-tests/sanity-functional-protractor/specs/api-spec.js

```diff
@@ -0,0 +1,121 @@
+
+        it('JSON Schema validation', () => {
+            const parametersSuiteName = testDataJson.get_metadata_by_suite_name.parameters;
+            const jsonSchemaPath = require(testDataJson.get_metadata_by_suite_name.json_schema_path);
+            return AetsApi.getMetadata(parametersSuiteName).then((response) => {
+                chakramExpect(response).to.have.status(HTTP_200_OK);
+                chakramExpect(response).to.have.schema(jsonSchemaPath);
+            });
+        });
+
+        it('Error message validation', () => {
+            const parametersSuiteName = testDataJson.get_metadata_by_suite_name.parameters;
+            const parametersNoExistingSuiteName = clone(parametersSuiteName);
+            parametersNoExistingSuiteName.suite = "no_existing_suite"
+            const errorMessageJson = testDataJson.get_metadata_by_suite_name.error_message;
+            return AetsApi.getMetadata(parametersNoExistingSuiteName).then((response) => {
+                chakramExpect(response).to.have.status(HTTP_400_BAD_REQUEST);
+                jasmineExpect(response.body).toEqual(errorMessageJson);
+            });
+        });
+    });
+    describe('Get metadata by correlationId - validation', () => {
+
+        it('JSON Schema validation', () => {
+            const parametersSuiteName = testDataJson.get_metadata_by_suite_name.parameters;
+            const parametersCorrelationId = testDataJson.get_metadata_by_correlationId.parameters;
+            const jsonSchemaPath = require(testDataJson.get_metadata_by_correlationId.json_schema_path);
+            return AetsApi.getMetadata(parametersSuiteName).then((response) => {
+                parametersCorrelationId.correlationId = response.body.correlationId;
+            }).then(() => {
+                return AetsApi.getMetadata(parametersCorrelationId).then((response) => {
+                    chakramExpect(response).to.have.status(HTTP_200_OK);
+                    chakramExpect(response).to.have.schema(jsonSchemaPath);
+                    jasmineExpect(response.body.correlationId).toEqual(parametersCorrelationId.correlationId);
+                });
+            });
+        });
+
+        it('Error message validation', () => {
+            const parametersCorrelationId = clone(testDataJson.get_metadata_by_correlationId.parameters);
+            parametersCorrelationId.correlationId = "no_existing_correlationId"
+            const errorMessageJson = testDataJson.get_metadata_by_correlationId.error_message;
+            return AetsApi.getMetadata(parametersCorrelationId).then((response) => {
+                chakramExpect(response).to.have.status(HTTP_400_BAD_REQUEST);
+                jasmineExpect(response.body).toEqual(errorMessageJson);
+            });
+        });
+    });
+
+    describe('Get artifact by artifact Id - validation', () => {
+
+        it('Status validation', () => {
+            const parametersSuiteName = testDataJson.get_metadata_by_suite_name.parameters;
```

Comment:
Mixed naming conventions, your using Camel Case and Snake Case.

Reply:
true, fixed